docs/installing.rst

```diff
@@ -102,15 +102,10 @@ On Windows [5]_:
 Python and OS Compatibility
 ---------------------------
 
-pip works with CPython versions 2.6, 2.7, 3.2, 3.3, 3.4, 3.5 and also pypy.
+pip works with CPython versions 2.6, 2.7, 3.3, 3.4, 3.5 and also pypy.
 
 pip works on Unix/Linux, OS X, and Windows.
 
-.. note::
-
-  Python 2.5 was supported through v1.3.1, and Python 2.4 was supported through
-  v1.1.
-
```

Comment:
We weren't keeping it up to date (for instance, it didn't include Python 3.1 being dropped) and it's sort of a pain (for instance, right now I'd say "Python 3.2 was supported through 7.1.2", but if we release 7.1.3 then this needs updated). It also duplicates information that is already available in the change log.